index.html

```diff
@@ -0,0 +1,33 @@
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>simpleweb</title>
+    </head>
+    <body>
+        <header>
+        <div class="toolbar">
+            <div class="logo">My Logo</div>
```

Comment:
Good job! but usually, the logo is a link to the home page, so please make sure to put this in the `a` tag.